scripts/setup/add_ros_repository.sh

```diff
@@ -24,3 +24,5 @@ sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31
 # Gazebo stuff
```

Comment:
Please use ubuntu keyserver like beofre with new key id (more trustworthy than downloading from random site) and replace the old key. Also no need to add curl to dependencies.

Reply:
Indeed I added `curl` to the deps list.

I'm not sure what the apt-key command should look like. I see that this page, http://wiki.ros.org/Installation/Ubuntu/Sources, appears to have the outdated key.

Reply:
Oh, I misread, you said no need to add curl. I thought it failed, but maybe I mis-recall. Let me try...

Reply:
I tried again, and `curl` needed to be installed for the `curl` command to work.

That http://wiki.ros.org/Installation/Ubuntu/Sources page does seem to be outdated, and I am waiting for them to reply with an update.

Reply:
I am saying to use wget which is already installed and used in the previous line instead of curl. They do the same thing.

You also need to remove the old key installation - C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654

Reply:
Alright, the ROS team responded, and they said that the original command should still work. I think the issue was I updated the `Dockerfile` at a point _after_ that `apt-key` command, so its result remained cached. `docker build --no-cache` is one way to build fresh again.

Trying a rebuild from scratch with the original command now.